substrate/pkg/controller/substrate/cluster/addons/tekton.go

```diff
@@ -35,11 +35,11 @@ func (t *Tekton) Create(ctx context.Context, substrate *v1alpha1.Substrate) (rec
 		return reconcile.Result{}, fmt.Errorf("initializing client, %w", err)
 	}
 	for _, file := range []string{
-		"https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.33.2/release.yaml",
-		"https://storage.googleapis.com/tekton-releases/triggers/previous/v0.19.0/release.yaml",
-		"https://github.com/tektoncd/dashboard/releases/download/v0.24.1/tekton-dashboard-release.yaml",
+		pipeline,
```

Comment:
I don't like this workaround. Without using helm chart, there is no good approach to override the definition in the yaml files without doing some YAML manipulation. It will take us some time to do it. Maybe in the later version.

Meanwhile, the Tekton helm charts's version is behind the current release, and make it not suitable for our use cases.

Reply:
I don't think we should go with this approach here and I would skip Tekton for now as we don't have this limitation with tekton to run on specific nodes with right role attached, we can taint the control plane nodes and either Tekton will run on management node or Karpenter can provision another node.

Reply:
Done